operators/src/mock/mock_raster_point_join.rs

```diff
@@ -91,23 +92,40 @@ impl VectorOperator for MockRasterPointJoinOperator {
         self: Box<Self>,
         context: &dyn crate::engine::ExecutionContext,
     ) -> Result<Box<crate::engine::InitializedVectorOperator>> {
-        InitializedOperatorImpl::create(
+        let vector_sources = self
+            .vector_sources
+            .into_iter()
+            .map(|o| o.initialize(context))
+            .collect::<Result<Vec<_>>>()?;
+
+        let raster_sources = self
+            .raster_sources
+            .into_iter()
+            .map(|o| o.initialize(context))
+            .collect::<Result<Vec<_>>>()?;
+
+        let result_descriptor = {
+            let mut columns = vector_sources[0].result_descriptor().columns.clone();
+            columns.insert(self.params.feature_name.clone(), FeatureDataType::Number);
```

Comment:
check if column with that name already exists?

Reply:
1b9537d